config/fair-queue.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | FairQueue Route Middleware
+    |--------------------------------------------------------------------------
+    |
+    | These middleware will get attached onto each FairQueue route, giving you
+    | the chance to add your own middleware to this list or change any of
+    | the existing middleware. Or, you can simply stick with this list.
+    |
+    */
+    'middleware' => ['web'],
```

Comment:
should be this:

<img width="777" alt="Screen Shot 2022-03-28 at 7 15 45 AM" src="https://user-images.githubusercontent.com/1913404/160417778-f8624587-e6d1-43ce-925c-2f630fac839f.png">